Factoriser adresse mail automatique (noreply)

Eskimon opened the pull request:
| Q | R |
| --- | --- |
| Correction de bugs ? | oui |
| Nouvelle Fonctionnalité ? | non |
| Tickets concernés | #629 |

Factorisation de l'adresse email de noreply du site.
Elle est dorénavant dans le settings.py
Un message d'avertissement est rajouté pour la modification

Ajout également d'un test pour vérifier l'envoi du mail de demande d'adhésion à l'assoc'

Eskimon committed:
Ajout d'un test pour l'adhésion à l'association - #627

Ajout d'un test pour vérifier que l'envoi d'un mail a bien lieu
lors de la demande d'adhesion à l'association.

Eskimon committed:
Factorisation de l'adresse de noreply du site - #629

Factorisation de l'adresse email de noreply du site.
Elle est dorénavant dans le settings.py
Un message d'avertissement est rajouté pour la modification

cgabard commented on zds/member/forms.py:
```diff
@@ -159,15 +161,6 @@ def clean(self):
-                del cleaned_data['password']
-            if 'password_confirm' in cleaned_data:
-                del cleaned_data['password_confirm']
-
```
Pourquoi virer ça ?

Eskimon commented:
Beuh c'est quoi ce bordel !! J'ai merdé entre deux branches on dirait car ca c'est pas censé disparaitre ici :/

Put**\* faut vraiment que je sois plus vigilant :/

cgabard commented:
C'est pas specialement grave, elles servent à ça les PR

Eskimon committed:
Annulation de modif sur un fichier non concerné - #629

Eskimon commented:
Voila qui est mieux !

cgabard commented on zds/settings.py:
```diff
@@ -304,6 +304,11 @@
 
 MAIL_CA_ASSO = 'ca-zeste-de-savoir@googlegroups.com'
 
+# ATTENTION, CETTE ADRESSE NE DOIT PAS ETRE CHANGÉE SANS ACCORD DU CA
```
Peux tu rajouter "de l'association Zds" a la fin ?

Quelqu'un qui tombe là dessus ne va pas forcément faire le liens entre CA et Conseil d'Admin de l'asso

Alex-D replied:
J'irais même jusqu'à mettre Conseil d'administration, en toutes lettres.

firm1 replied:
Dans le settings.py (comme dans tous les .py) il me semble qu'il faudrait
écrire les commentaires en anglais.
Le 2 juin 2014 17:40, "Alexandre Demode" notifications@github.com a écrit
:

> In zds/settings.py:
> 
> > @@ -304,6 +304,11 @@
> > 
> >  MAIL_CA_ASSO = 'ca-zeste-de-savoir@googlegroups.com'
> > 
> > +# ATTENTION, CETTE ADRESSE NE DOIT PAS ETRE CHANGÉE SANS ACCORD DU CA
> 
> J'irais même jusqu'à mettre Conseil d'administration, en toutes lettres.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zestedesavoir/zds-site/pull/670/files#r13290876.

Eskimon committed:
Petit update pour clarifier une phrase - #629

Petit update pour clarifier une phrase d'informations
et suppression du backend de debug (oubli)

poulp commented:
La variable `MAIL_NOREPLY` ne serait elle pas mieux placée dans le fichier `settings_prod` ? Ainsi que de mettre une valeur spéciale pour la version dev. Vu que c'est quelque chose de sensible à la modification il est préférable de la mettre dans un fichier non servi par git et modifiable uniquement par ceux qui ont un accès au serveur.

Eskimon committed:
Diverses bricoles esthetiques - #629

Eskimon commented:
Bon, apparemment Travis passe pas certains test qui passait chez moi... Je vérifie, met d'equerre et vous tiens au courant (comme quoi, un fix meme facile ne l'est pas pour tout le monde :D )

Eskimon committed:
Correction d'un test et setting en anglais - #629

firm1 commented:
> La variable MAIL_NOREPLY ne serait elle pas mieux placée dans le
> fichier settings_prod ?

Toutes les variables du settings peuvent être modifiées dans le
settings_prod. Donc ce n'est pas si grave.
Le 2 juin 2014 17:54, "poulp" notifications@github.com a écrit :

> La variable MAIL_NOREPLY ne serait elle pas mieux placée dans le fichier
> settings_prod ? Ainsi que de mettre une valeur spéciale pour la version
> dev. Vu que c'est quelque chose de sensible à la modification il est
> préférable de la mettre dans un fichier non servi par git et modifiable
> uniquement par ceux qui ont un accès au serveur.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zestedesavoir/zds-site/pull/670#issuecomment-44855292
> .

cgabard commented:
Oui mais il a pas tord : elle devrait être  redéfinit dans le setting de
prod, par sécurité
Le 2 juin 2014 18:42, "firm1" notifications@github.com a écrit :

> > La variable MAIL_NOREPLY ne serait elle pas mieux placée dans le
> > fichier settings_prod ?
> 
> Toutes les variables du settings peuvent être modifiées dans le
> settings_prod. Donc ce n'est pas si grave.
> Le 2 juin 2014 17:54, "poulp" notifications@github.com a écrit :
> 
> > La variable MAIL_NOREPLY ne serait elle pas mieux placée dans le fichier
> > settings_prod ? Ainsi que de mettre une valeur spéciale pour la version
> > dev. Vu que c'est quelque chose de sensible à la modification il est
> > préférable de la mettre dans un fichier non servi par git et modifiable
> > uniquement par ceux qui ont un accès au serveur.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/zestedesavoir/zds-site/pull/670#issuecomment-44855292>
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zestedesavoir/zds-site/pull/670#issuecomment-44861141
> .

Alex-D commented:
+1 pour le @poulp :)

firm1 commented:
> Oui mais il a pas tord : elle devrait être  redéfinit dans le setting de
> prod, par sécurité

Ouaip, mais je veux dire, là ça devient la tambouille de celui qui déploie zds, donc, la PR en elle même est bonne.

Alex-D commented:
A ce moment là, le commentaire est à retirer :)
